doc/guidelines/frontend-ui-usability.md

```diff
@@ -0,0 +1,195 @@
+  Instead, make the unit part of the input field itself or leave it out if it's clear from the context.
+- **Submission** to the backend should not happen until all fields are **verified** by the **frontend** logic.
+- In case of submissions **on-the-fly**, e.g. setting dates for plants, the data should always be submitted with **debouncing**.
+- **Double submission** should be **prevented** by disabling the submission button until the server has responded.
+
+### Consistency
+
+Things with the **same meaning** should **look and behave** the **same** throughout the whole application.
+Users should never be uncertain if different words mean the same thing or different actions trigger the same process.
+
+Examples:
+
+- **Main Heading** of a page is always defined by `<h1>` which must be the only one of its kind per page
+- **Headings** of all toolbars are always defined by `<h2>`
+- If **textual links**, e.g. the links in PermaplanT's navigation bar, need to be surrounded by space, always set their `margin` instead of `padding`.
+- Make **call-to-action buttons** look the same everywhere by using our `SimpleButton` component.
+  Use Tailwind's classes to define the button's `margin` and `width`, if necessary due to layout or viewport.
+- **Toast errors** are always shown on the top right.
+  To achieve this, we are simply using the default setting of our used library (_react-toastify_).
+- Use the same **application-specific terms** everywhere when communicating to the user, e.g. always write _PermaplanT_.
+- **Forms** and their fields should look and behave the same everywhere according to our guidelines in [Forms](#forms).
+- In every **form**, **validation** errors are detected and shown by using the native input validation of HTML5.
+
+### Icons
+
+- A **tooltip** on hovering must be displayed for every icon in PermaplanT's toolbox.
+  The tooltip contains the icon's **label** (as concise as possible) and, if existing, the assigned **shortcut**.
+- When **active**, i.e. currently activated/enabled, they must get a small **highlighting** through a visually stronger and more colorful appearance by using the design's primary colors.
+- When **passive**, i.e. currently not active/enabled, they must must appear in **neutral** colors/colorless.
+- When **disabled**, i.e. currently not clickable, they must be **greyed-out**, via decreasing their opacity.  
+  The mouse cursor must be styled with the `not-allowed` css property.
+
+### Highlighting
+
+Highlighting techniques:
+
+- Font weight:
+
+  - **Bold** text adds tolerable noise to the design and enables distinct highlighting in both short and long texts.
+    In general, it is the preferred highlighting technique to use.
+  - _Italic_ text adds minimal noise to the design, but lacks in recognizability.
+  - <u>Underlining</u> adds most noise and compromises a text's legibility.
+    It should not be used.
+
+- Colors: primary and secondary colors of the design's color palette
```

Comment:
I personally think blue is not the best color for text highlighting, because of the resemblance to hyperlinks.